packages/azure-themes/src/azure/styles/DefaultButton.styles.ts

```diff
@@ -58,19 +58,24 @@ export const DefaultButtonStyles = (theme: ITheme): Partial<IButtonStyles> => {
       border: `${StyleConstants.borderWidth} solid ${extendedSemanticColors.primaryButtonBorderDisabled} !important`,
     },
     rootFocused: {
+      selectors: {
+        '::after': {
+          outlineColor: `${extendedSemanticColors.ButtonBorderFocus} !important`,
+        },
+      },
       backgroundColor: semanticColors.buttonBackground,
       color: semanticColors.buttonText,
       fill: semanticColors.buttonTextHovered,
       border: `${StyleConstants.borderWidth} solid ${semanticColors.inputBorder}`,
     },
     rootHovered: {
-      border: `${StyleConstants.borderWidth} solid ${semanticColors.inputBorderHovered}`,
+      // border: `${StyleConstants.borderWidth} solid ${semanticColors.inputBorderHovered}`,
```

Comment:
Hey @aidanmc95 , everything looks good. Let's just remove these 2 comments and I'll merge in. Same goes for the cherry-picked version. :)